### ISSUE TITLE
Doesn't work in herbstluftwm's configs

### ISSUE THREAD

**Issue by EliteGuzhva** [altdesktop/playerctl#245]
**Description**
Recently I have updated playerctl to the latest version (from 2.0.2) in order to use playerctld daemon.
_Everything works fine from the command line_. Daemon start as a systemd service.
When calling playerctl commands from herbstluftwm (v0.9.2) scripts (for keybindings and polybar) I get **error-code 127**. However ```playerctl --version (--help)``` works as expected. Any other command, such as ```status, play, etc``` exit with an error.

**What I've tried**
- changing Player (playerctld, spotify, firefox, chromium). Didn't help
- starting herbstluftwm with ```dbus-run-session herbstluftwm``` and ```dbus-launch herbstluftwm```. Neither worked for me (DBUS_SESSION_BUS_ADDRESS did change)
- printing running players in polybar from dbus (```dbus-send --session --dest=org.freedesktop.DBus --type=method_call --print-reply /org/freedesktop/DBus org.freedesktop.DBus.ListNames```) and playerctl (```playerctl --list-all```). Both outputs are correct.
- enabling debug logging. Haven't seen any logs.

**Comment by acrisci** [comment]
I don't believe I can do anything in the code to help this. I think the problem is with how your system is set up.

> When calling playerctl commands from herbstluftwm (v0.9.2) scripts (for keybindings and polybar) I get error-code 127

This exit code means the binary is not found or a library dependency is not found. This script might be altering your `PATH` env variable to find the binary. Try using an absolute path such as `/usr/bin/playerctl play` to run commands. Try to get stdout to find any error messages.

**Comment by EliteGuzhva** [comment]
@acrisci The problem is that the binary is in PATH. ```whereis playerctl``` returns exactly ```/usr/bin/playerctl```. And I can get the output of ```playerctl --help``` or ```playerctl --version```. 
Right now the workaround is to install the latest version, launch the daemon and then reinstall the old version (v2.0.2). It works correctly with the daemon.
Is there maybe a way to install playerctld separately? It fails when I'm trying to launch it with v2.0.2.

**Comment by EliteGuzhva** [comment]
Note: Old version is installed via apt
